tracing/src/lib.rs

```diff
@@ -869,6 +869,13 @@ pub mod __macro_support {
     pub type Once = tracing_core::Once;
 }
 
+#[doc(hidden)]
+// resolves https://github.com/tokio-rs/tracing/issues/783 by forcing a monomorphization
+// in tracing, not downstream crates.
+pub fn is_enabled(meta: &crate::Metadata<'_>) -> bool {
```

Comment:
This should probably be 
```suggestion
#[inline]
pub fn is_enabled(meta: &crate::Metadata<'_>) -> bool {
```
since it's only ever called from downstream crates, and we can't get cross-crate inlining without an annotation. We _may_ even want `inline(always)` here, but I'd run the benchmarks first...

Reply:
I'll start with an `#[inline]`; we can always change this later.